internal/pkg/cli/validate.go

```diff
@@ -231,6 +233,13 @@ func basicNameValidation(val interface{}) error {
 }
 
 func validateCron(sched string) error {
+	// If the schedule is wrapped in aws terms `rate()` or `cron()`, don't validate it--
+	// instead, pass it in as-is for serverside validation. AWS cron is weird (year field, nonstandard wildcards)
+	// so for edge cases we need to support it
+	awsSchedMatch := awsScheduleRegexp.FindStringSubmatch(sched)
```

Comment:
Maybe we can create a `manifest/schedule` package with a `ValidateExpr(expr string) error` function, that has the common functionality between `cli/validate.go` and `stack/scheduled_job.go`.

Probably worthwhile if you think there are other neat things we can put in the `schedule` package

Reply:
I think for the time being I'll leave it as is, but I could see for instance an `manifest/events` package for jobs which handles the logic for parsing a yaml description of a CW Event Rule based on arbitrary event patterns, which could also include schedule validation.